drafts/draft-20005.adoc

```diff
@@ -0,0 +1,225 @@
+
+  <connectionFactory id="cf1" jndiName="eis/cf1" jaasLoginContextEntryRef="CTGEntry">
+    <properties.eciResourceAdapter ConnectionUrl="tcp://localhost" portNumber="2006" serverName="MYSERVER"/>
+  </connectionFactory>
+
+  ...
+</server>
+----
+
+The same approach can be used for JAAS Custom Login Modules that are packaged within an application. Set the `classProviderRef` to point to the `id` of the `application`, `webApplication`, or `enterpriseApplication` element that contains the login module class. When packaging JAAS Custom Login Modules within an application, include the login module within one of the following places:
+
+** Within a top level JAR of the enterprise application.
+** Within a resource adapter module of the enterprise application.
+** Within the web module of the enterprise application.
+** Within an EJB module of the enterprise application.
+** Within a web application.
+
+It should be noted that JAAS Custom Login Modules require the use of a resource reference with container managed authentication.
+
+You can find out more about link:https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/twlp_sec_jaas.html[Configuring a JAAS custom login module for Liberty].
+
+[#OLL]
+=== Open Liberty console logging now has the ability to format logs with date and time stamps and other relevant information
+
+In Liberty, users have the option to format their server logs that appear in the console.log file in different formats, such as JSON and dev (default) formats, in their server logging configuration, using the consoleFormat logging attribute. The dev format will show messages in a basic format, where there are no timestamp or any other relevant information. It will only show the message with its log level and the message itself.
+
+For example:
+[source, xml]
+----
+consoleFormat=dev (default)
+[AUDIT ] CWWKE0001I: The server server1 has been launched.
+----
+
+This feature introduces a new option called `simple` for the consoleFormat logging server configuration attribute. This new option will configure Liberty to output logs in the same simple (default) format as the message.log file, with date/time stamps and other relevant information, to the `console.log` file or to the console (Standard-out).
+
+For example:
+
+[source, xml]
+----
+consoleFormat=simple
+[25/11/19 10:02:30:080 EST] 00000001 com.ibm.ws.kernel.launch.internal.FrameworkManager A CWWKE0001I: The server server1 has been launched.
+----
+
+To configure the Liberty logs to output logs in the new simple console format, you just have to set the following logging server configuration, in the bootstrap.properties, server.xml, or in the server.env (environment variables).
+
+==== server.env
+`WLP_LOGGING_CONSOLE_FORMAT=simple`
+
+==== bootstrap.properties
+`com.ibm.ws.logging.console.format=simple`
+
+==== server.xml
+`<logging consoleFormat="simple"/>`
```

Comment:
I'd put these three in the same order as they're listed in the preceding sentence.

Reply:
Alasdair wanted them in a specific order with server.xml first. I have changed the sentence to match the list.